server/game/cards/02_SHD/upgrades/ArmedToTheTeeth.ts

```diff
@@ -0,0 +1,27 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { UpgradeCard } from '../../../core/card/UpgradeCard';
+import { RelativePlayer, WildcardCardType } from '../../../core/Constants';
+
+export default class ArmedToTheTeeth extends UpgradeCard {
+    protected override getImplementationId() {
+        return {
+            id: '1938453783',
+            internalName: 'armed-to-the-teeth',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addGainOnAttackAbilityTargetingAttached({
+            title: 'Give another friendly unit +2/+0 for this phase',
+            targetResolver: {
+                cardTypeFilter: WildcardCardType.Unit,
+                controller: RelativePlayer.Self,
```

Comment:
Need to add the following property so that the ability can't target the attached unit (it says "another unit"):

```typescript
cardCondition: (card, context) => card !== context.source,
```